listener.go

```diff
@@ -214,6 +222,7 @@ func insertDevelopers(rows []common.Row, txn *sql.Tx) bool {
 		ele.Get("username", &Username)
 		ele.Get("first_name", &FirstName)
 		ele.Get("last_name", &LastName)
+		ele.Get("email", &Name)
```

Comment:
I had added it to make it homogeneous for the query, while I was testing UNION. Its not needed now. Removed it. Done.